scapy/layers/inet6.py

```diff
@@ -396,7 +402,7 @@ def route(self):
         return conf.route6.route(dst)
 
     def mysummary(self):
-        return "%s > %s (%i)" % (self.src,self.dst, self.nh)
+        return "%s > %s (%i)" % (str(self.src),str(self.dst), self.nh)
```

Comment:
`self.src` and `self.dst` calls packet with `str()` in case they are `Net6` objects